src/System.Private.ServiceModel/tests/Common/Scenarios/Endpoints.cs

```diff
@@ -183,4 +184,12 @@ public static string Tcp_CustomBinding_SslStreamSecurity_Address
             return BridgeClient.GetResourceAddress("WcfService.TestResources.TcpTransportSecurityWithSslResource");
         }
     }
+
+    public static string Tcp_CustomBinding_SslStreamSecurity_HostName
+    {
+        get
+        {
+            return BridgeClient.GetResourceHostName("WcfService.TestResources.TcpTransportSecurityWithSslResource");
+        }
```

Comment:
i agree with you. Currently I'm following how #457 does its naming, so at least it's consistent there. We can take this consistency issue on once both are merged I guess?